docs/MAINTAINERS.md

```diff
@@ -1,52 +1,187 @@
+- Commit the changes with the version number as the commit title, including the `v` prefix, to `master`. For example:
 
-- Commit the changes with the version number as the commit title, including the
-`v` prefix, to `master`. For example:
+**NOTE:** The version **MUST** be prefixed with a "v"
 
-```sh
+```bash
 git commit -m "v1.0.0" # not 1.0.0
 ```
 
-- Create an annotated tag for the new version. The commit title should equal
-the annotated tag name. For example:
+- Create an annotated tag for the new version. The commit title should equal the annotated tag name. For example:
 
-```sh
+```bash
 git tag -a v1.0.0 -m "v1.0.0"
 ```
 
 - Push the commit and the annotated tag.
 
-```sh
+```bash
 git push
 git push --tags
 ```
 
-Upgrading Electron
-------------------
+- Open a pull request against `master` titled "Release v1.0.0"
+
+### Generating binaries
+
+**Environment**
+
+Make sure to set the analytics tokens when generating production release binaries:
+
+```bash
+export ANALYTICS_SENTRY_TOKEN="xxxxxx"
+export ANALYTICS_MIXPANEL_TOKEN="xxxxxx"
+```
+
+#### Linux
+
+##### Clean dist folder
+
+**NOTE:** Make sure to adjust the path as necessary (here the Etcher repository has been cloned to `/home/$USER/code/etcher`)
+
+```bash
+./scripts/build/docker/run-command.sh -r x64 -s . -c "make distclean"
+```
+
+##### Generating artefacts
+
+```bash
+# x64
+
+# Build Debian packages
+./scripts/build/docker/run-command.sh -r x64 -s . -c "make electron-develop && make RELEASE_TYPE=production electron-installer-debian"
+# Build RPM packages
+./scripts/build/docker/run-command.sh -r x64 -s . -c "make electron-develop && make RELEASE_TYPE=production electron-installer-redhat"
+# Build AppImages
+./scripts/build/docker/run-command.sh -r x64 -s . -c "make electron-develop && make RELEASE_TYPE=production electron-installer-appimage"
+# Build CLI
+./scripts/build/docker/run-command.sh -r x64 -s . -c "make electron-develop && make RELEASE_TYPE=production cli-installer-tar-gz"
+
+# x86
+
+# Build Debian packages
+./scripts/build/docker/run-command.sh -r x86 -s . -c "make electron-develop && make RELEASE_TYPE=production electron-installer-debian"
+# Build RPM packages
+./scripts/build/docker/run-command.sh -r x86 -s . -c "make electron-develop && make RELEASE_TYPE=production electron-installer-redhat"
+# Build AppImages
+./scripts/build/docker/run-command.sh -r x86 -s . -c "make electron-develop && make RELEASE_TYPE=production electron-installer-appimage"
+# Build CLI
+./scripts/build/docker/run-command.sh -r x86 -s . -c "make electron-develop && make RELEASE_TYPE=production cli-installer-tar-gz"
+```
 
-- Upgrade the `electron` dependency version in `package.json` to an *exact
-  version* (no `~`, `^`, etc).
+#### Mac OS
+
+**ATTENTION:** For production releases you'll need the code-signing key,
+and set `CSC_NAME` to generate signed binaries on Mac OS.
+
+**NOTE:** The CLI is not code-signed for either at this time.
```

Comment:
What does "either" mean here?

Reply:
Oh, looks like there's a "platform" missing after that "either"